src/Mutation/Item.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Softonic\GraphQL\Mutation;
+
+use Softonic\GraphQL\Traits\JsonSerializer;
+
+class Item implements MutationObject, \JsonSerializable
+{
+    use JsonSerializer;
+
+    private $arguments;
+
+    private $config;
+
+    private $hasChanged = false;
+
+    public function __construct(array $arguments = [], array $config = [], bool $hasChanged = false)
+    {
+        $this->arguments  = $arguments;
+        $this->config     = $config;
+        $this->hasChanged = $hasChanged;
+    }
+
+    public function __get(string $key)
+    {
+        if (is_null($this->arguments[$key]) && array_key_exists($key, $this->config)) {
```

Comment:
What happens if that key contains the value `null`?

Reply:
If there is no key in config.... that just returns null? Is it right?

Reply:
Exactly, it won't enter to the condition.